src/test/java/io/personium/core/model/CellTest.java

```diff
@@ -0,0 +1,140 @@
+                return null;
+            }
+
+            @Override
+            public String getDataBundleName() {
+                return null;
+            }
+
+            @Override
+            public void makeEmpty() {
+
+            }
+            @Override
+            public void delete(boolean recursive, String unitUserName) {
+
+            }
+
+            @Override
+            public List<Role> getRoleListForAccount(String username) {
+                return null;
+            }
+
+            @Override
+            public List<Role> getRoleListHere(IExtRoleContainingToken token) {
+                return null;
+            }
+            @Override
+            public String roleIdToRoleResourceUrl(String roleId) {
+                return null;
+            }
+            @Override
+            public String roleResourceUrlToId(String roleUrl, String baseUrl) {
+                return null;
+            }
+
+        };
+        testCell.id = CELL_ID;
+        testCell.name = CELL_NAME;
+        testCell.owner = CELL_NAME;
+    }
+
+    @AfterClass
+    public static void tearDownAfterClass() throws Exception {
+        PersoniumUnitConfig.reload();
+        CommonUtils.setFQDN(fqdnShelter);
+    }
+
+    @Test
+    public void getUrl() {
+        PersoniumUnitConfig.set(PersoniumUnitConfig.PATH_BASED_CELL_URL_ENABLED, "true");
+        String result = testCell.getUrl();
+        log.info(result);
+        assertEquals("https://testunit.example/testcell/", result);
+
+        PersoniumUnitConfig.set(PersoniumUnitConfig.PATH_BASED_CELL_URL_ENABLED, "false");
+        result = testCell.getUrl();
+        log.info(result);
+        assertEquals("https://testcell.testunit.example/", result);
+    }
+
+    @Test
+    public void getUnitUrl_Returns_UnitUrl_RegardlessOfCellUrlModes() {
+        PersoniumUnitConfig.set(PersoniumUnitConfig.PATH_BASED_CELL_URL_ENABLED, "true");
+        String result = testCell.getUnitUrl();
+        assertEquals("https://testunit.example/", result);
+        PersoniumUnitConfig.set(PersoniumUnitConfig.PATH_BASED_CELL_URL_ENABLED, "false");
+        String result2 = testCell.getUnitUrl();
+        assertEquals(result, result2);
+    }
+
+    @Test
+    public void getOwnerNormalized() {
+        // When just a normal string
+        testCell.owner = "abcde";
+        // Should be same as raw
+        assertEquals(testCell.getOwnerRaw(), testCell.getOwnerNormalized());
+
+        // When http based url
+        testCell.owner = "https://example.com/";
+        // Should be same as raw
+        assertEquals(testCell.getOwnerRaw(), testCell.getOwnerNormalized());
+
+        // When personium-localunit schem url
+        testCell.owner = "personium-localunit:cellname:/";
+        String result = testCell.getOwnerNormalized();
+        log.info(result);
+        // Should be resolved to http based url
+        assertEquals(UriUtils.convertSchemeFromLocalUnitToHttp(testCell.getOwnerRaw()), result);
```

Comment:
This test code should be fixed to either one.
- Make a mock for UriUtils.convertSchemeFromLocalUnitToHttp, and assert it is invoked
- Use raw value like 'http://cellname.unit/' as expected value